tests/Makefile.am

```diff
@@ -33,7 +34,8 @@ src_liblogger_la_CFLAGS = $(BUILD_FLAGS)
 # TESTS
 ################################################
 TESTING_FLAGS = -I system_config -I ../src -I include \
-                $(WARNING_FLAGS) $(CPPUNIT_CFLAGS)
+                -std=c++11 \
```

Comment:
I think this will work with g++ 4.6 (precise) . It Certainly works in 4.8 which is in trusty.  Debian Squeeze comes with 4.8